frontend/src/mockData/db.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
```

Comment:
Since this is part of the API, which for instance is a fake one, let's move this into a new folder: /backend/mock/db.json.

Reply:
Done